si2400.md

```diff
@@ -5,25 +5,15 @@ Name: Sayef Iqbal
 UNI: si2400
 
 Github: [link](https://github.com/sayefiqb)
+Github README: [link](https://github.com/sayefiqb/sayefiqb/blob/main/README.md)
 
 
 ## Project Proposal
-1. API Healthcheck dashboard (something like pingdom)
-2. Application version page (lists information about application, e.g. docker container image tag)
-3. ormar (CRUDS to MySQL and PostgreSQL using API endpoints)
-4. Tool to convert from json to string and vice versa
-5. Library tool to count, analyze and visualize AWS infrastructure in json/csv by region
-6. Tool to encrypt data or file (base64) or some other enncoding mechanism and save to s3
-7. Create random data in json format by sepcifying data types and common column names
+[Proposal](https://github.com/sayefiqb/project-proposals-s2023/blob/hw1_si2400/projects/python/kms-encryption-tool.md)
 
 
 ## Project Contributions
-1. [numpy] https://github.com/numpy/numpy/issues/22977
-2. [slack] https://github.com/slackapi/python-slack-sdk/issues/1313
-3. [slack] https://github.com/slackapi/python-slack-sdk/issues/1322
-4. [twilio] https://github.com/twilio/twilio-python/issues/646
-5. [kubernetes] https://github.com/kubernetes-client/python/issues/1957
-6. [kubernetes] https://github.com/kubernetes-client/python/issues/1961
-7. [botocore] https://github.com/boto/botocore/issues/2742
-8. [botocore] https://github.com/boto/botocore/issues/2723
-
+[Kubernetes](https://github.com/sayefiqb/project-proposals-s2023/blob/hw1_si2400/projects/python/kubernetes.md)
```

Comment:
use relative links

Reply:
I've pushed new changes to this PR. Can I use same PR?